packages/react-app/src/components/global/Header/DAOSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react';
```

Comment:
Allows toggling of DAO using drop down menu. Currnetly Selected Customer is passed across components using the React Context API